Core/Mapper15.h

```diff
@@ -33,7 +37,8 @@ class Mapper15 : public BaseMapper
 				bank |= subBank;
 				SelectPRGPage(0, bank);
 				SelectPRGPage(1, bank + 1);
-				SelectPRGPage(2, bank + ((addr & 0x02) ? 0 : 1));
+				bank = ((addr & 0x02) ? bank : 0xFE) | subBank;
```

Comment:
sorry for my error, but since line 37 already says " bank |= subBank ", the extra " | subBank " here is unnecesary.